.github/workflows/tests.yml

```diff
@@ -65,3 +65,11 @@ jobs:
       - name: yarn test:consumer
         if: matrix.node-version == '14.x'
         run: yarn test:consumer
+
+      - name: install bf-cli
+        if: matrix.node-version == '14.x'
```

Comment:
I've been trying to "stripe" tasks across the different versions - effectively "load balancing" them. Are these still distributed well across the 3 versions?

Reply:
Probably not. I went for "put them all on the most recent", but I'll check and load balance.